.github/workflows/build_conda_pkg.yml

```diff
@@ -6,10 +6,10 @@ on:
       - main
   pull_request:
     types: [ opened, synchronize ]
-    paths:
-      - 'requirements.txt'
-      - 'core-requirements.txt'
-      - 'evalml/tests/dependency_update_check/*.txt'
+#    paths:
```

Comment:
Was this intentional?

Reply:
It’s just so that I could trigger the job. Will revert prior to merge!